### ISSUE TITLE
Lyra2RE3 - no pools available

### ISSUE THREAD

**Issue by sionprawn** [MultiPoolMiner/MultiPoolMiner#2421]
Hello U-Guru,

Really sorry if I'm wasting your, genuinely valuable time.  I did read your comment on the Lyra2RE3, no pools available issue (which is minor to me), but from what I read I can only conclude there is an issues with the available "Lyra2Rev3" Pools.

I have 2 machines, 1 Nvid and 1 AMD, both get the same result if I just try to mine just "Vertcoin" (for testing perposess) which has only 1 coin related to the "lyra2Rev3" Algo.  So when I tested just for this particular Coin/Alog (of which there is only 1 coin)  I get "No Pools Available" (see attached TXT file)

So when;

"MPM polls all configured pools and builds a list of all available pool/algo possibilities (http://localhost:3999/allpools.html).
* MPM selects the best paying pool for each algorithm (http://localhost:3999/bestpools.html)
* MPM scans the files in the MinersLegacy folder and builds a list of all available miner/algo combinations (http://localhost:3999/miners.html) thereby evaluating the GPUs capabilities, e.g. min. RAM required for each algo, and pool/algo availabilty in allpools
* MPM selects the fastest miner for each algorithm (non-benchmarked miner/algos are always selected first) from bestpools and builds a list (http://localhost:3999/activeminers.html)
* MPM calculates the best paying miner per GPU (miner speed * price) and starts mining.

and it finds "No Pools available" when the only Algo chosen is used for Vertcoin, is this a fault with the "Lyra2R3 MPM Pool", or some other Pool.

[MultiPoolMiner_2019-09-29_05-55-56.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/3666387/MultiPoolMiner_2019-09-29_05-55-56.txt)

1/ I have included only 1 pool/algo possibility
2/ The best paying Pool (it finds No Pools)
3/ I have the available Miners and a 1080ti (+another one) on one machine and a Vega 64 on the other + stacks of RAM on both, so should be able to mine this algo (I used to be able to anyway)
4/ As for it selecting the fastest miner for each Algo,  I've checked and have the required miners, so it should choose whichever one it deems fit.
5/ As to calcutating the best paying miner, I don't know, as it never benchmarks any of them due to not finding a Pool.

In short, is this down to, Pools that are out of your control, or am I just a dimwit (my guess is dimwit).

**Comment by UselessGuru** [comment]
@ sionprown

Thank you for your detailed problem description.
Please try this:

https://raw.githubusercontent.com/MultiPoolMiner/MultiPoolMiner/9e264d545991edab8d67b667e3742c31c3722cf4/Algorithms.txt

**Comment by sionprawn** [comment]
Cheers, 

I tried that, and it did benchmark L...3, this time, But this time each miner came up with the error, "MPM could not retrieve hash rate information from the miner API within between 45-60 seconds"

I amended the "MultiPoolMiner.ps1" benchmakring for 60s to 120s but got the same reasults.

[Test_MultiPoolMiner_lyra2RE3_2019-09-29.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/3667298/Test_MultiPoolMiner_lyra2RE3_2019-09-29.txt)

**Comment by UselessGuru** [comment]
@sionprawn 

There appears an issue with MPH, port 17027 is not responding - I've sent them a support message.
For now just disable mining on MiningPoolHub-Algo (MiningPoolHub-Coin is fine)

**Comment by sionprawn** [comment]
Cheers U-Gu

**Comment by UselessGuru** [comment]
[Fixed with 3.5.3](https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/v3.5.3)
